chanzuckerberg/single-cell-data-portal
# fix(cell-cards): DAG Label Click Fix

## Reason for Change

The way we're handling the text label now (clicking the text label can collapse/expand a node) messes up the node centering operation when a CellCard is cached in memory. I can chat more about this with you in a huddle because it's hard to explain over text
- text labels should only collapse/expand node for dummy nodes

## Changes

- add
- remove
- modify
    - Moving handleClick listener

## Testing steps

- https://localhost:3000/cell-guide/CL_0000540
- Click on labels in table, ensure that cell type names and "n cell types" expand/collapse when clicking on the text

## Notes for Reviewer
